test/model/TestModelTest.php

```diff
@@ -233,7 +233,7 @@ public function testGetConfig(){
             ->disableOriginalConstructor()
             ->setMethods(array('getOnePropertyValue'))
             ->getMock();
-        $propInstanceContent = new \core_kernel_classes_Property(TEST_TESTCONTENT_PROP);
+        $propInstanceContent = new \core_kernel_classes_Property(TestService::TEST_TESTCONTENT_PROP);
```

Comment:
all TEST_TESTCONTENT_PROP are deprecated, replace it, please

Reply:
Same case here, just replace not really usages